web-marketplace/src/pages/ProfileEdit/ProfileEdit.tsx

```diff
@@ -96,10 +96,7 @@ export const ProfileEdit = () => {
             <ViewProfileButton setIsWarningModalOpen={setIsWarningModalOpen} />
           )}
         </Flex>
-        <WithLoader
-          isLoading={!loaded || accountChanging || loading}
-          sx={{ mx: 'auto' }}
-        >
+        <WithLoader isLoading={accountChanging || loading} sx={{ mx: 'auto' }}>
```

Comment:
wallet loaded state is changing while connecting to keplr which causes the profile edit outlet to reload
but we don't really need it now since we already check for auth loading, and only logged in user can access those pages.